lib/utils/fs.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+
+const path = require('path');
```

Comment:
we can use import here, e.g. https://github.com/paulirish/pwmetrics/blob/master/lib/outh/google-outh.ts#L5